### PR TITLE
{2023.06,2023a,zen4} Highway v1.0.4 with fix for failing test

### PR DESCRIPTION
**Pull request by trz42**
This is mainly for testing if a patch provided in https://github.com/easybuilders/easybuild-easyconfigs/pull/20942 fixes a failing test. See https://github.com/EESSI/software-layer/pull/613

### CODE REVIEW

**Comment by trz42** [comment]
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen4

**Comment by trz42** [comment]
Build again without the patch...

bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen4

**Comment by boegel** [comment]
@trz42 Seems like patch is doing its job?

**Comment by trz42** [comment]
> @trz42 Seems like patch is doing its job?

Yes. Looks like it works. Only tested for zen4 though.

**Comment by trz42** [comment]
Build again after updating the patch file (description only) and easyconfig (checksum for patch file only)...

bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen4

**Comment by boegel** [comment]
@trz42 Since https://github.com/easybuilders/easybuild-easyconfigs/pull/20942 is merged, this is ready to deploy?

File: easystacks/software.eessi.io/2023.06/zen4/eessi-2023.06-eb-4.9.2-2023a.yml [hunk 1]
```diff
@@ -11,3 +11,8 @@ easyconfigs:
       options:
         # see https://github.com/easybuilders/easybuild-easyconfigs/pull/20958
         from-commit: dbadb2074464d816740ee0e95595c2cb31b6338f
+  #- Highway-1.0.4-GCCcore-12.3.0.eb
+  - Highway-1.0.4-GCCcore-12.3.0.eb:
+      options:
+        # see https://github.com/easybuilders/easybuild-easyconfigs/pull/20942
+        from-pr: 20942
```
Comment (ocaisa) [inline easystacks/software.eessi.io/2023.06/zen4/eessi-2023.06-eb-4.9.2-2023a.yml:18]:
```suggestion
        # see https://github.com/easybuilders/easybuild-easyconfigs/pull/20942
        from-commit: 524da37b903585cea5a9eeb4156d1c8d57636bd8
```

**Comment by ocaisa** [comment]
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen4

File: easystacks/software.eessi.io/2023.06/zen4/eessi-2023.06-eb-4.9.2-2023a.yml [hunk 2]
```diff
@@ -11,3 +11,8 @@ easyconfigs:
       options:
         # see https://github.com/easybuilders/easybuild-easyconfigs/pull/20958
         from-commit: dbadb2074464d816740ee0e95595c2cb31b6338f
+  #- Highway-1.0.4-GCCcore-12.3.0.eb
```
Comment (ocaisa) [inline easystacks/software.eessi.io/2023.06/zen4/eessi-2023.06-eb-4.9.2-2023a.yml:14]:
```suggestion
```

**Review by ocaisa — APPROVED** [review]

**Pull request merged**